app/app/src/test/java/com/globant/calculator/android/mvp/presenter/CalculatorPresenterTest.java

```diff
@@ -0,0 +1,249 @@
+package com.globant.calculator.android.mvp.presenter;
+
+import com.globant.calculator.android.mvp.model.CalculatorModel;
+import com.globant.calculator.android.mvp.view.CalculatorView;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static com.globant.calculator.android.utils.Constants.DIVIDE;
+import static com.globant.calculator.android.utils.Constants.DOT_BUTTON;
+import static com.globant.calculator.android.utils.Constants.EMPTY_STRING;
+import static com.globant.calculator.android.utils.Constants.MINUS;
+import static com.globant.calculator.android.utils.Constants.MULTIPLY;
+import static com.globant.calculator.android.utils.Constants.NUMBER_ZERO;
+import static com.globant.calculator.android.utils.Constants.PLUS;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class CalculatorPresenterTest {
```

Comment:
try to put all the initializations together and all the assertions and validations together.. if not the test is very difficult to read.

Reply:
done, thanks.